tests/scripts/test_generate_terraform.py

```diff
@@ -444,6 +444,45 @@ def test_dataset_tf_file_contains_description_when_specified(
         ) == 1
 
 
+def test_pipeline_tf_contains_bq_table_description_when_specified(
```

Comment:
Is there a test for an invalid description? (is there such thing?)

Reply:
and for no description?

Reply:
@leahecole Invalid descriptions such as incorrect string syntax will be covered by the `test_validation_*` tests in `tests/scripts/test_generate_terraform.py`

As for those with no descriptions, yes we should add tests for those! Here you go: 3c5ac43 and 534c409